coral/README.md

```diff
@@ -0,0 +1,87 @@
+# 🪸 Klaw's new frontend app
+
+- Please be aware of our [Code of Conduct](../CODE_OF_CONDUCT.md) ❤️
+
+## About
+
+`/coral` contains a React app. It's the rewrite of the existing Klaw frontend. 
+
+## Installation and usage 
+
+ℹ️ Coral uses `pnpm` as a package manager. Read in their official documentation [how to install](https://pnpm.io/installation) it. 
+
+- navigate to this directory
+- run `pnpm install`
+- run `pnpm dev` to start the frontend app in development mode
+
+### Usage: How to run the app
+
+ℹ️ You can see all our scripts in the [`package.json`](package.json).
+You can also run `pnpm` in your console to get a list of all available scripts.
```

Comment:
`pnpm` outputs the subcommands of the tool itself. I think you meant to use `pnpm run` here that describes the different commands we have defined for the project.

Reply:
🙈  yes! Thanks for paying close attention 🙏